test/e2e/network/ingress.go

```diff
@@ -153,6 +154,29 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			// framework.ExpectNoError(jig.verifyURL(fmt.Sprintf("https://%v/", ip), "", 30, 1*time.Second, httpClient))
 		})
 
+		It("multicluster ingress should get instance group annotation", func() {
+			name := "echomap"
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "http"), ns, map[string]string{
+				"kubernetes.io/ingress.class": "gce-multi-cluster",
```

Comment:
Maybe make these consts as well?

Reply:
Humm, seems we have one for the key:
https://github.com/kubernetes/kubernetes/blob/4c8539cece2f0a6e6974b30d00c7341e10320bc5/test/e2e/framework/ingress_utils.go#L65

Reply:
Done.
Also added a TODO to reuse these consts from ingress-gce instead of duplicating them here